app/views/shared/_accounts_table.html.haml

```diff
@@ -7,5 +7,4 @@
     - @accounts.each do |account|
       %tr
         %td= payment_source_link_or_text(account)
-        %td{style: "padding-left: 50px"}
-          = account.facility || "<i>All</i>".html_safe
+        %td= account.per_facility? ? account.facilities.join(", ") :  content_tag(:i, t("shared.all"))
```

Comment:
Is there no better way in HAML to write HTML than `content_tag`? This reads kinda yucky.

Reply:
To do it in line, I think that's the best way. The other option would be something like this.
```haml
%td
  - if account.per_facility?
    = account.facilities.join(", ")
  - else
    %i= t("shared.all")
```

Reply:
We could also do something like `html("shared.all")` and throw some markdown in the locale.

Reply:
Nah, all of those seem quite gross, so I’m okay with it as is :) Thanks!